docs/operations/comanage-recipes.md

```diff
@@ -0,0 +1,71 @@
+COManage Recipes
+==============================
+
+A collection of step-by-step instructions for common actions for use by administrators of the OSG COManage.
+
+
+Steps for Common COManage Actions
+----
+This section contains some common actions administrators perform on the OSG COManage, and how to do so.
```

Comment:
```suggestion
```

All of this basically just repeats what you say in the sentence above. Also, adding a section level here adds noise to the section hierarchy and it's not entirely clear why we need it (if we have another section in this doc at the same level, what would it actually be?)